onnxruntime/python/tools/transformers/models/stable_diffusion/README.md

```diff
@@ -74,90 +74,116 @@ Below is an example to optimize Stable Diffusion 1.5 in Linux. For Windows OS, p
+You can also follow the [official docs](https://onnxruntime.ai/docs/build/eps.html#amd-rocm) to build with docker.
 
 ### Export ONNX pipeline
+This step will export stable diffusion 1.5 to ONNX model in float32 using script from diffusers.
 
-This step will export stable diffusion 1.5 to ONNX model in float32 using script from diffusers. Before running the script, you need to be logged in via `huggingface-cli login`.
+It is recommended to use PyTorch 1.12.1 or 1.13.1 in this step. Using PyTorch 2.0 will encounter issue in exporting onnx.
+
+Before running the script, you need to be logged in via `huggingface-cli login`.
 
 ```
 curl https://raw.githubusercontent.com/huggingface/diffusers/v0.15.1/scripts/convert_stable_diffusion_checkpoint_to_onnx.py > convert_sd_onnx.py
-python convert_sd_onnx.py --model_path runwayml/stable-diffusion-v1-5  --output_path  ./sd-v1-5
+python3 convert_sd_onnx.py --model_path runwayml/stable-diffusion-v1-5  --output_path  ./sd_onnx_fp32
 ```
 
 ### Optimize ONNX Pipeline
 
 Example to optimize the exported float32 ONNX models, and save to float16 models:
 
 ```
-python -m onnxruntime.transformers.models.stable_diffusion.optimize_pipeline -i ./sd-v1-5 -o ./sd-v1-5-fp16 --float16
+python3 -m onnxruntime.transformers.models.stable_diffusion.optimize_pipeline -i ./sd_onnx_fp32 -o ./sd --float16
 ```
 
 If you installed ONNX Runtime v1.14, some optimizations (packed QKV and BiasAdd) will be disabled automatically since they are not available in v1.14.
 
-For Stable Diffusion 2.1 model with CUDA EP, you will need force Attention to run in float32 to avoid black image by appending `--force_fp32_ops unet:Attention` to the command line.
-If you are using nightly package, append `--force_fp32_ops unet:MultiHeadAttention` instead.
-
 ### Run Benchmark
 
 The benchmark.py script will run a warm-up prompt twice, and measure the peak GPU memory usage in these two runs, then record them as first_run_memory_MB and second_run_memory_MB. Then it will run 5 runs to get average latency (in seconds), and output the results to benchmark_result.csv.
 
 Note that the first run might need more time and memory: For example, cuDNN convolution algorithm search or model compile happens in the first run.
 
+To avoid black image output for Stable Diffusion 2.1 with CUDA EP, we can set an environment variable before inferencing:
+```
+export ORT_DISABLE_TRT_FLASH_ATTENTION=1
+```
+
 Example to benchmark the optimized pipeline with batch size 1 on CUDA EP:
 ```
-python -m onnxruntime.transformers.models.stable_diffusion.benchmark -p ./sd-v1-5-fp16/ -b 1
+python3 -m onnxruntime.transformers.models.stable_diffusion.benchmark -p ./sd -b 1
 ```
 
 On ROCm EP, use the following command instead:
 ```
-python -m onnxruntime.transformers.models.stable_diffusion.benchmark -p ./sd-v1-5-fp16/ -b 1 --tuning --provider=rocm
+python3 -m onnxruntime.transformers.models.stable_diffusion.benchmark -p ./sd -b 1 --tuning --provider rocm
```

Comment:
Do we really need to emphasize `python3` and `pip3`, `python` and `pip` should work pretty well nowadays.